pkg/controller/direct/compute/forwardingrule_controller.go

```diff
@@ -161,6 +161,16 @@ func (m *forwardingRuleModel) AdapterForObject(ctx context.Context, reader clien
 			}
 			obj.Spec.Target.TargetSSLProxyRef.External = targetSSLProxyRef.External
 		}
+
+		// Get target TCPProxy
```

Comment:
Does it mean this is an optional field and if not provided the compute service will assign a default? Can we add a test suite for this? no blocker, can be done in follow up PR